src/backend/distributed/utils/citus_clauses.c

```diff
@@ -452,3 +461,40 @@ CitusIsMutableFunction(Node *node)
 
 	return false;
 }
+
+
+/* FixFunctionArguments applies expand_function_arguments to all function calls. */
```

Comment:
Maybe:

`FixFunctionArguments converts named-notation args to positional args
 and/or inserts default args, as needed in all function calls in the expr node tree.`